static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -744,6 +744,11 @@ class Crossword extends Component {
                 this.grid = grid;
             },
         };
+        // Trigger the custom event when component has loaded for ad slot in commercial
+        useEffect(() => {
+            const customEvent = new CustomEvent('crossword-loaded', {detail: {source: 'frontend_bundle'}});
```

Comment:
I don't think we're using `detail`? If we need it in the future, we can add it then.